pom.xml

```diff
@@ -143,13 +143,13 @@
     <profile>
       <id>jackson-2.12</id>
       <properties>
-        <jackson.version>2.12.6</jackson.version>
+        <jackson.version>2.12.7</jackson.version>
       </properties>
     </profile>
     <profile>
-      <id>jackson-2.13.1</id>
+      <id>jackson-2.13</id>
       <properties>
-        <jackson.version>2.13.1</jackson.version>
+        <jackson.version>2.13.4</jackson.version>
```

Comment:
2.13.5 was released in late Jan 2023, so maybe update to use the latest 2.13.x

Reply:
```suggestion
<jackson.version>2.13.5</jackson.version>
```